internal/apps/dop/component-protocol/components/issue-manage/issueTable/render.go

```diff
@@ -240,9 +243,34 @@ type AssigneeOperationData struct {
 }
 
 type ComponentAction struct {
-	labels  []apistructs.ProjectLabel
-	isGuest bool
-	userMap map[string]string
+	labels   []apistructs.ProjectLabel
+	isGuest  bool
+	userMap  map[string]string
+	state    State
+	issueSvc pb.IssueCoreServiceServer
+	ctx      context.Context
```

Comment:
[Prefer contexts passed as arguments](https://go.dev/blog/context-and-structs) , not a field in struct